### PR TITLE
once_cell

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
Hmm. Looks like `regex!` macro needs rust 1.45

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# console
+[![crates.io](https://img.shields.io/crates/d/console.svg)](https://crates.io/crates/console)
+[![crates.io](https://img.shields.io/crates/v/console.svg)](https://crates.io/crates/console)
+[![Documentation](https://docs.rs/console/badge.svg)](https://docs.rs/console)
+[![Continuous Integration](https://github.com/mitsuhiko/console/workflows/Continuous%20Integration/badge.svg)](https://github.com/mitsuhiko/console/actions)
```
Comment (pksunkara) [inline README.md:4]: I don't like to have these.

**Review by pksunkara — CHANGES_REQUESTED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (burrbull) [reply README.md:4]: All this, or just last?

Comment (pksunkara) [reply README.md:4]: All this please

Comment (burrbull) [reply README.md]: Done.
But why? It is helpfull.

Comment (pksunkara) [reply README.md]: I observed that it increases maintenance burden however infinitesimal it maybe.

**Review by pksunkara — APPROVED** [review]

**Pull request merged**